raiden/raiden_service.py

```diff
@@ -677,6 +680,48 @@ class RaidenEventHandler(object):
 
     def __init__(self, raiden):
         self.raiden = raiden
+        self.event_listeners = list()
+
+    def start_event_listener(self, event_name, filter_, translator):
```

Comment:
Oh I see. I totally forgot that it's a reserved kyword in python. Makes sense then.